integration-tests/bats/helper/local-remote.bash

```diff
@@ -0,0 +1,163 @@
+load helper/query-server-common
+
+SKIP_SERVER_TESTS=$(cat <<-EOM
```

Comment:
Are we going to treat this as a burndown? I'm assuming this is all files with the exception of a few. Can you add a comment which describes what's in this test please.

Reply:
Yes. This is a burndown. Once every dolt command in a bats file is migrated, we can remove that file from this list and all the tests in that file will automatically be run in both modes.

In practice, most of these files test a single command and then use a couple of other commands (sql, status, diff) to either setup the test environment or examine the results. So once these "helper" commands are migrated it should be one-for-one with the subsequent commands.